### PR TITLE
feat:designed the mobile view

### PR DESCRIPTION
**Pull request by Macdeal**

### CODE REVIEW

File: airbnb/index.html [hunk 1]
```diff
@@ -468,58 +468,301 @@ <h3>Become a host</h3>
...
+            <path
+              d="m16 28c7-4.733 14-10 14-17 0-1.792-.683-3.583-2.05-4.95-1.367-1.366-3.158-2.05-4.95-2.05-1.791 0-3.583.684-4.949 2.05l-2.051 2.051-2.05-2.051c-1.367-1.366-3.158-2.05-4.95-2.05-1.791 0-3.583.684-4.949 2.05-1.367 1.367-2.051 3.158-2.051 4.95 0 7 7 12.267 14 17z">
+            </path>
+          </svg>
+        </div>
+      </div>
+    </div>
+
+    <div class="card-elements">
+      <div class="name-and-rating">
```
Comment (yoki1234) [inline airbnb/index.html:588]: Check :  padding on each child element [as it is notice able differencs]

File: airbnb/index.html [hunk 2]
```diff
@@ -139,25 +139,25 @@ <h3>Where to?</h3>
               <div class="discover-card">
                 <div class="img">
                   <img src="assets/img/n13.jpg" alt="n13" />
-                 </div>
+                </div>
                 <p> exploring</p>
               </div>
               <div class="discover-card">
```
Comment (yoki1234) [inline airbnb/index.html:145]: Fix: There is bottom border of 2px, while clicking on it.
in every discover-card

**Review by yoki1234 — APPROVED** [review]
AIRBNB

Review yourself before sending PR.
--CodeSpeed

File: airbnb/index.html (hunk 2 above)
Comment (Macdeal) [reply airbnb/index.html:145]: ok

**Review by Macdeal — COMMENTED** [review]
resolved

File: airbnb/index.html (hunk 2 above)
Comment (yoki1234) [reply airbnb/index.html:145]: Is it ready for merging?

**Pull request merged**